### PR TITLE
Migrate to ws (continue)

### PR DESCRIPTION
**Pull request by SkeLLLa**
Continue of PR #34 

#### Checklist

- [X] run `npm run test` and `npm run benchmark`
- [X] tests and/or benchmarks are included
- [x] documentation is changed or added
- [X] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

Fixed tests.

This potentially introduces breaking change. As `ws`'s `createWebSocketStream` returns node.js stream without any additional props that were in `websocket-stream` I've replaced old `connection` with object that contains two properties: `stream` - websocket stream and `socket` which refers to websocket instance. Actually it's possible just to add socket to node stream object that `createWebSocketStream` returns, but I'd rather prefer to keep it as is.

@mcollina, @Eomm please review and tell if you're okay with such change or not. And after that I continue with updating docs and probably rewriting `verifyClient`, cause it's deprecated and may be removed soon.

PS: Current ws version doesn't support node 6, and I think that this plugins should also drop it's support.

### CODE REVIEW

**Comment by mcollina** [comment]
> Actually it's possible just to add socket to node stream object that createWebSocketStream returns, but I'd rather prefer to keep it as is.

I would prefer to add another property to our stream object as `websocket-stream`  does.

> PS: Current ws version doesn't support node 6, and I think that this plugins should also drop it's support.

I'm ok in dropping Node 6 in this PR. 

> And after that I continue with updating docs and probably rewriting verifyClient, cause it's deprecated and may be removed soon.

Where is verifyClient deprecated?

**Comment by SkeLLLa** [comment]
> I would prefer to add another property to our stream object as websocket-stream does.
Ok, will do.

> Where is verifyClient deprecated?
https://github.com/websockets/ws/issues/377#issuecomment-462152231

File: index.js [hunk 1]
```diff
@@ -62,7 +63,8 @@ function fastifyWebsocket (fastify, opts, next) {
 
   function handleRouting (connection, request) {
     const response = new ServerResponse(request)
-    request[kWs] = connection
+    request[kWs] = WebSocket.createWebSocketStream(connection)
+    request[kWs].socket = connection
```
Comment (Eomm) [inline index.js]: Oh!! That's why my PR wasn't working 😆
Good catch 💪🏻

Comment (SkeLLLa) [reply index.js]: Now I understand why you've reverted some of your code :).

**Pull request merged**